sql/core/src/main/scala/org/apache/spark/sql/execution/streaming/FileStreamSourceLog.scala

```diff
@@ -36,8 +33,8 @@ class FileStreamSourceLog(
     path: String)
   extends CompactibleFileStreamLog[FileEntry](metadataLogVersion, sparkSession, path) {
 
-  import CompactibleFileStreamLog._
-  import FileStreamSourceLog._
+  import org.apache.spark.sql.execution.streaming.CompactibleFileStreamLog._
```

Comment:
Why is it necessary to change to import with the full name here?

Reply:
maybe changed by Idea. I have reverted